src/DataObjects/Request.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Dredd\DataObjects;
+
+class Request
+{
+    /** @var string $body */
+    public $body;
+
+    /**
+     * Can be manually set in {@link https://dredd.org/en/latest/hooks/index.html#hooks hooks}
+     *  - `utf-8` (string) - indicates `body` contains a textual content encoded in UTF-8
+     *  - `base64` (string) - indicates `body` contains a binary content encoded in Base64
+     *
+     * @var string $bodyEncoding
+     * @psalm-var 'utf-8'|'base64' $bodyEncoding
+     */
+    public $bodyEncoding;
```

Comment:
I don't see where this is defined in the dredd docs [here](https://github.com/apiaryio/dredd-transactions#transaction-object) and I also don't see it on https://dredd.org/en/latest/hooks/index.html#hooks. Can you point out where this is documented? It seems to me the Dredd docs specify all the properties except `bodyEncoding`.

Reply:
It's in the data structures page.

https://dredd.org/en/latest/data-structures.html#transaction-object

Under "request" there it is.